specification/purview/data-plane/Azure.Purview.Catalog/preview/2020-12-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,6748 @@
+        "operationId": "Entity_CreateOrUpdate",
+        "x-ms-examples": {
+          "EntityREST_Create": {
+            "$ref": "./examples/EntityREST_Create.json"
+          },
+          "EntityREST_Update": {
+            "$ref": "./examples/EntityREST_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "entity",
+            "description": "Atlas entity with extended information.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AtlasEntityWithExtInfo"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersionAtlas"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Created with EntityMutationResponse.",
+            "schema": {
+              "$ref": "#/definitions/EntityMutationResponse"
+            }
+          }
+        }
+      }
+    },
+    "/atlas/v2/entity/bulk": {
+      "get": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "List entities in bulk identified by its GUIDs.",
+        "description": "List entities in bulk identified by its GUIDs.",
+        "operationId": "Entity_GetByGuids",
+        "x-ms-examples": {
+          "EntityREST_GetByGuids": {
+            "$ref": "./examples/EntityREST_GetByGuids.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "guid",
+            "in": "query",
+            "description": "An array of GUIDs of entities to create.",
+            "required": true,
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi"
+          },
+          {
+            "$ref": "#/parameters/minExtInfo"
+          },
+          {
+            "$ref": "#/parameters/ignoreRelationships"
+          },
+          {
+            "$ref": "#/parameters/excludeRelationshipTypes"
+          },
+          {
+            "$ref": "#/parameters/apiVersionAtlas"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/AtlasEntitiesWithExtInfo"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "Create or update entities in bulk.",
+        "description": "Create or update entities in Atlas in bulk.\nExisting entity is matched using its unique guid if supplied or by its unique attributes eg: qualifiedName.\nMap and array of collections are not well supported. E.g., array<array<int>>, array<map<string, int>>.",
+        "operationId": "Entity_CreateOrUpdateBulk",
```

Comment:
This should probably be `Entity_BulkCreateOrUpdate` to match with the `BulkDelete` operation.

Reply:
Rename BulkDelete to Deletebyguids to follow Atlas behavior.